source/expressions.tex

```diff
@@ -4222,7 +4222,7 @@
 then let \tcode{T1} be the type of \tcode{E1}.
 \tcode{E2} shall designate either
 a member of \tcode{T1} or
-a direct base class relationship $(\tcode{T1}, \tcode{B})$.
```

Comment:
The parens are maths, so why not keep this as is?

Reply:
Because I messed up when reverting the change.